pubsub/subscriptions/subscription_test.go

```diff
@@ -746,7 +747,7 @@ func TestCreateWithFilter(t *testing.T) {
 	}
 }
 
-func TestCreateBigQuerySubscription(t *testing.T) {
+func TestBigQuerySubscription(t *testing.T) {
```

Comment:
Nit: why change this test name? CreateBigQuerySubscription seems to accurately describe what it's doing.

Reply:
I think I wanted to make this test do more than "create" but I'll revert for now